src/index.test.ts

```diff
@@ -304,6 +304,56 @@ describe('Running @erc725/erc725.js tests...', () => {
     });
   });
 
+  describe('Getting data (using new getDataBatch) in schema by provider [e2e] - luksoTestnet', () => {
+    const web3 = new Web3('https://rpc.testnet.lukso.network');
+    const ERC725_BATCH_CONTRACT_ADDRESS =
+      '0x4b30900F119E11D2A8CAe18176c4f9840E586Cc4';
+
+    const e2eSchema: any = [
```

Comment:
Why do you use `any` as a type? 

Ideally, we should never use any. It should be like very rare case and exceptions. Let's not take the habit of writing any too much.